src/Module/Objects.php

```diff
@@ -57,7 +57,7 @@ public static function rawContent(array $parameters = [])
 			['order' => ['origin' => true]]
 		);
 		// Valid items are original post or posted from this node (including in the case of a forum)
-		if (!DBA::isResult($item) || !$item['origin'] && !strstr($item['author-link'], DI::baseUrl()->get())) {
+		if (!DBA::isResult($item) || !$item['origin'] && (parse_url($item['author-link'], PHP_URL_HOST) != parse_url(DI::baseUrl()->get(), PHP_URL_HOST))) {
```

Comment:
What about subfolder Friendica installs?

Reply:
Since we only compare the host names we don't need to care for subfolders. And it is very unlikely that one does have two Friendica instances on the same system, just separated by the subfolders. And even this was the case, then it doesn't really matter, since the worst thing that can happen is that you return a post that wasn't from your system.